src/main.rs

```diff
@@ -246,18 +250,25 @@ async fn main() -> anyhow::Result<()> {
 
     let otel_config = OtelConfig {
         traces_exporter: args.otel_traces_exporter,
+        metrics_exporter: args.otel_metrics_exporter,
         exporter_otlp_endpoint: args.otel_exporter_otlp_endpoint,
     };
     let log_level = WasmcloudLogLevel::from(args.log_level);
+    let service_name = "wasmcloud-host".to_string();
+
     if let Err(e) = configure_tracing(
-        "wasmcloud-host".to_string(),
+        service_name.clone(),
         &otel_config,
         args.enable_structured_logging,
         Some(&log_level),
     ) {
         eprintln!("Failed to configure tracing: {e}");
     };
 
+    if let Err(e) = configure_metrics(&service_name, &otel_config) {
```

Comment:
I think this should be called from the `configure_tracing` logic (which we can rename to `configure_observability`)

Reply:
talked offline, we'll leave this change for the future unifying observability PR.